src/components/Register/index.tsx

```diff
@@ -32,13 +37,34 @@ export const Register = () => {
   } = useForm<RegisterForm>({
     resolver: zodResolver(RegisterFormSchema),
     defaultValues: DEFAULT_REGISTER_FORM_VALUES,
-  })
+  });
+
+  const onSubmit = async (data: RegisterForm) => {
+    try {
+      const participants = data.participants.map((participant) => ({
+        participant_name: participant.participant_name,
+        participant_country: participant.participant_country,
+        participant_email: participant.participant_email,
+        project_name: data.project_name,
+        project_description: data.project_description,
+        project_url: data.project_url,
+      }));
+
+      const { data: participantsData, error: participantsError } =
+        await supabase.from("Participants").insert(participants);
```

Comment:
Handle errors gracefully
```suggestion
        const {error, data} = await supabase.from("Participants").insert(participants);
        if (error) { // code here}
```